python/cuml/dask/common/base.py

```diff
@@ -18,27 +18,78 @@
 import numpy as np
 from toolz import first
 
-import cudf.comm.serialize  # noqa: F401
+from cuml.dask.common.utils import get_client
+from cuml.dask.common.input_utils import get_datatype
+import pickle
 
 from cuml import Base
 from cuml.common.array import CumlArray
 
 from dask_cudf.core import DataFrame as dcDataFrame
 
-from dask.distributed import default_client
 from functools import wraps
 
 
 class BaseEstimator(object):
 
-    def __init__(self, client=None, verbose=False, **kwargs):
+    def __init__(self, client=None, verbose=False, model=None, **kwargs):
         """
         Constructor for distributed estimators
         """
-        self.client = default_client() if client is None else client
+        self.client = get_client(client)
         self.verbose = verbose
         self.kwargs = kwargs
 
+        self.set_model(model)
+
+    @classmethod
+    def load(cls, file, client=None, verbose=False, pickle_args={}, **kwargs):
```

Comment:
Should probably add support for loading from a byte stream in addition to a file... I feel like every time we require a file, we end up wanting a stream too.